documentation/source/documentation/cheatsheets/python.rst

```diff
@@ -0,0 +1,600 @@
+|    [5, 10, 15]         |   #[5, 10, 15]                                      |
++------------------------+-----------------------------------------------------+
+
+.. _`multiple values`: https://opendylan.org/about/examples/multiple_return_values.html
+
+Syntax
+------
+
+.. note::
+
+   In Dylan, any words after an ``end`` (e.g.  ``end method``) are
+   optional.
+
++----------------------------------+------------------------------------------+
+| Python                           | Dylan                                    |
++==================================+==========================================+
+| .. code-block:: python           | .. code-block:: dylan                    |
+|                                  |                                          |
+|    var = exp                     |    define variable var = exp;            |
+|    VAR = exp # Convention        |    define variable var :: type = exp;    |
+|                                  |    define constant var = exp;            |
++----------------------------------+------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                    |
+|                                  |                                          |
+|    f(x, y, z)                    |    f(x, y, z)                            |
++----------------------------------+------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                    |
+|                                  |                                          |
+|    1                             |    begin 1; 2; 3; end                    |
+|    2                             |    begin 1; 2; 3 end                     |
+|    3                             |                                          |
++----------------------------------+------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                    |
+|                                  |                                          |
+|    lambda x, y, *z:              |    method (x, y, #rest z)                |
+|      print('hello')              |      say("hello");                       |
+|      return f(x, y, z)           |      f(x, y, z)                          |
+|                                  |    end method                            |
++----------------------------------+------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                    |
+|                                  |                                          |
+|    x = 5                         |    begin                                 |
+|    body                          |      let x = 5;                          |
+|                                  |      body                                |
+|                                  |    end                                   |
```

Comment:
This is a bit unfair to Dylan. If your intention is to show how to create a local scope, then on the Python side I would just say "The function is the smallest scope".  The only way to do it (unless there's something new in Python 3?) is to define a local function and call it.